### ISSUE TITLE
Templated strings

### ISSUE THREAD

**Issue by SergioMorchon** [palantir/tslint#332]
Hi.

When linting this function:
``` typescript
function makeHeader(pkg: Package) {
	return [
		`\t/**`,
		`\t * ${pkg.name} v${pkg.version}`,
		`\t */`
	].join("\r\n");
}
```

i got the message ```jsdoc is not formatted correctly on this line``` in 2nd (``` `\t * ${pkg.name} v${pkg.version}`,```) and 3th (``` `\t */` ```) lines that are missinterpreted as comments.

**Comment by arnaudpourbaix** [comment]
It is the same issue I reported yesterday: see https://github.com/palantir/tslint/issues/331
It is very annoying because I like template strings but linter will report many issues and even VS doesn't properly display template strings content (especially when I use " or ' inside).

**Comment by adidahiya** [comment]
@leeavital, can you please add a regression test case for this in the jsdoc rule?
